user.py

```diff
@@ -0,0 +1,5 @@
+class User:
+    def __init__(self, name, password, email):
+        self.name = name
+        self.password = password
+        self.email = email
```

Comment:
add empty line

Reply:
There is also an empty line at the end of file. i.e. there are 6 lines in this file I don't know why it doesn't appear on your side.